float32 m_maxX
float32 m_maxY
float32 m_maxZ
float32 m_resolution
---
int32[] shape
int32[] semantic_grid